test/functional/specs/Visitor/C36908.js

```diff
@@ -35,12 +36,26 @@ const setConsent = ClientFunction(consent => {
   return window.alloy("setConsent", consent);
 });
 
-test("C36908 When ID migration is enabled and Visitor and Alloy are both awaiting consent, when Visitor is denied and Alloy is approved, an error occurs.", async () => {
+const getIdentity = ClientFunction(() => {
+  return window.alloy("getIdentity");
+});
+
+test("C36908 When ID migration is enabled and Visitor and Alloy are both awaiting consent, when Visitor is denied and Alloy is approved, Alloy gets an ECID which is the same as Visitor's.", async () => {
   await createMockOptIn(false);
+
   await configureAlloyInstance("alloy", config);
   let errorMessage;
   await setConsent(CONSENT_IN).catch(err => {
     errorMessage = err.errMsg;
```

Comment:
I'm guessing `errMsg` was a legacy opt-in thing? It might make more sense at this point to just remove the `.catch` here, which should fail the test if any error is thrown.